servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/GraphQLConfigurationProperties.kt

```diff
@@ -45,6 +45,11 @@ data class GraphQLConfigurationProperties(
          */
         val enabled: Boolean = false,
 
+        /**
+         * Boolean flag indicating whether we want to generate Federation v2 compatible schema.
+         */
+        val optInV2: Boolean = true,
```

Comment:
should this be `false` by default ?, documentation says that.

Reply:
yes it should